common/hooks/pre-configure/02-script-wrapper.sh

```diff
@@ -92,7 +92,9 @@ export PKG_CONFIG_LIBDIR="$XBPS_CROSS_BASE/usr/lib/pkgconfig\${PKG_CONFIG_LIBDIR
 exec /usr/bin/pkg-config "\$@"
 _EOF
 	chmod 755 ${XBPS_WRAPPERDIR}/${XBPS_CROSS_TRIPLET}-pkg-config
-	ln -sf ${XBPS_CROSS_TRIPLET}-pkg-config ${XBPS_WRAPPERDIR}/pkg-config
+	if [ -z $no_generic_pkgconfig_link ]; then
```

Comment:
Does this work without  `$no_generic_pkgconfig_link` enclosed in quotes? That could be  `if [ -z ]; then`... not sure if bash is happy with that :)

Reply:
`if [ -z $a ] ; then echo x ; else echo y ; fi` dash, bash and zsh print 'x' and don't throw syntax error. i guess you are referring to the numerical comparison operators, which indeed fail if a var is empty and not quoted

Reply:
I should quote it nevertheless, since `no_generic_pkgconfig_link="some reason"` would get split and blow up the test.